src/no-status-code-assert.ts

```diff
@@ -0,0 +1,104 @@
+// no-status-code-assert.ts
+
+/*
+ * Copyright (c) 2022-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import { AST_NODE_TYPES, ESLintUtils, TSESLint, TSESTree } from '@typescript-eslint/utils';
+import getDocumentationUrl from './get-documentation-url';
+
+export const ruleId = 'no-status-code-assert';
+const NO_STATUS_CODE_ASSERT = 'NO_STATUS_CODE_ASSERT';
+const keywords = ['status', 'code', 'StatusCodes', 'statusCode'];
```

Comment:
`status`/`code` are widely used in none response status code related assertions, e.g. things like `account.status`/`message.code` can be incorrectly labelled.

Reply:
@le-cong refactored to just check for StatusCodes for all assertions in parameters else if it is BinaryExpression check on either side.